### ISSUE TITLE
definition is void: copy-list

### ISSUE THREAD

**Issue by hostmaster** [sellout/emacs-color-theme-solarized#136]
Hi
I'm using Emacs 25.0.50 (9.0) on Mac with color-theme-solarized-20150107.743 from melpa. 
I've got an error "definition is void: copy-list"  and with `--debug-init` the tracelog is 

```
Debugger entered--Lisp error: (void-function copy-list)
  copy-list((:foreground base0 :background back))
  solarized-face-for-index((:foreground base0 :background back) 1)
  create-face-spec(default (:foreground base0 :background back))
  apply(create-face-spec (default (:foreground base0 :background back)))
  #[(face) "\301\302\"\207" [face apply create-face-spec] 3]((default (:foreground base0 :background back)))
  mapcar(#[(face) "\301\302\"\207" [face apply create-face-spec] 3] ((default (:foreground base0 :background back)) (cursor (:foreground base03 :background base0)) (error (:weight bold :slant normal :underline nil :inverse-video nil :foreground red)) (escape-glyph-face (:foreground red)) (fringe (:foreground base01 :background base02)) (linum (:weight normal :slant normal :underline nil :inverse-video nil :foreground base01 :background base02)) (header-line (:foreground base0 :background base02 :weight normal :slant normal :underline nil :inverse-video t)) (highlight (:background base02)) (hl-line (:underline nil :background base02)) (isearch (:weight normal :slant normal :underline nil :inverse-video t :foreground orange :background back)) (isearch-fail (:weight normal :slant normal :underline nil :inverse-video t :foreground orange :background back)) (lazy-highlight (:weight normal :slant normal :underline nil :inverse-video t :foreground yellow :background back)) (link (:weight normal :slant normal :underline t :inverse-video nil :foreground violet)) (link-visited (:weight normal :slant normal :underline t :inverse-video nil :foreground magenta)) (match ((t (:weight normal :slant normal :underline nil :inverse-video t :foreground yellow :background back)))) (menu (:foreground base0 :background base02)) (minibuffer-prompt (:weight bold :slant normal :underline nil :inverse-video nil :foreground cyan)) (mode-line (:foreground base1 :background base02 :weight normal :slant normal :underline nil :inverse-video t :box nil)) (mode-line-inactive (:foreground base00 :background base02 :weight normal :slant normal :underline nil :inverse-video t :box nil)) (region (:foreground base01 :background base03 :weight normal :slant normal :underline nil :inverse-video t)) (secondary-selection (:background base02)) (shadow (:foreground base01)) (trailing-whitespace (:weight normal :slant normal :underline nil :inverse-video t :foreground red)) (vertical-border (:foreground base0)) (comint-highlight-prompt (:foreground blue)) (compilation-info (:weight bold :slant normal :underline nil :inverse-video nil :foreground green)) (compilation-warning (:weight bold :slant normal :underline nil :inverse-video nil :foreground orange)) (custom-button (:foreground base1 :background base02 :box (:line-width 2 :style released-button))) (custom-button-mouse (:weight normal :slant normal :underline nil :inverse-video t :foreground base1 :background base02 :inherit custom-button)) (custom-button-pressed (:weight normal :slant normal :underline nil :inverse-video t :foreground base1 :background base02 :box (:line-width 2 :style pressed-button) :inherit custom-button-mouse)) (custom-changed (:weight normal :slant normal :underline nil :inverse-video t :foreground blue :background base3)) (custom-comment (:foreground base1 :background base02)) (custom-comment-tag (:foreground base1 :background base02)) (custom-documentation (:inherit default)) (custom-group-tag (:foreground base1)) (custom-group-tag-1 (:weight bold :slant normal :underline nil :inverse-video nil :foreground base1)) (custom-invalid (:weight normal :slant normal :underline nil :inverse-video t :foreground red :background back)) (custom-link (:foreground violet)) (custom-state (:foreground green)) (custom-variable-tag (:foreground base1)) (diff-added (:weight bold :slant normal :underline nil :inverse-video nil :foreground green :background base02)) (diff-changed (:weight bold :slant normal :underline nil :inverse-video nil :foreground yellow :background base02)) (diff-removed (:weight bold :slant normal :underline nil :inverse-video nil :foreground red :background base02)) (diff-refine-change (:weight bold :slant normal :underline nil :inverse-video nil :foreground blue :background base02)) (diff-file-header (:background back)) (diff-header (:foreground base1 :background back)) (ido-only-match (:foreground green)) (ido-subdir (:foreground blue)) (ido-first-match (:weight bold :slant normal :underline nil :inverse-video nil :foreground green)) (emacs-wiki-bad-link-face (:weight normal :slant normal :underline t :inverse-video nil :foreground red)) ...))
  solarized-color-definitions()
  (apply (quote custom-theme-set-faces) (quote solarized-light) (solarized-color-definitions))
  eval-buffer()  ; Reading at buffer position 391
  load-theme(solarized-light t)
  eval-buffer(#<buffer  *load*> nil "/Users/igor/.emacs.d/init.el" nil t)  ; Reading at buffer position 1413
  load-with-code-conversion("/Users/igor/.emacs.d/init.el" "/Users/igor/.emacs.d/init.el" t t)
  load("/Users/igor/.emacs.d/init" t t)
  #[0 "\205\262	\306=\203\307\310Q\202;	\311=\204\307\312Q\202;\313\307\314\315#\203*\316\202;\313\307\314\317#\203:\320\nB\321\202;\316\322\323\322\211#\210\322=\203a\324\325\326\307\327Q!\"\323\322\211#\210\322=\203`\210\203\243\330!\331\232\203\243\332!\211\333P\334!\203}\211\202\210\334!\203\207\202\210\314\262\203\241\335\"\203\237\336\337#\210\340\341!\210\266\f?\205\260\314\323\342\322\211#)\262\207" [init-file-user system-type delayed-warnings-list user-init-file inhibit-default-init inhibit-startup-screen ms-dos "~" "/_emacs" windows-nt "/.emacs" directory-files nil "^\\.emacs\\(\\.elc?\\)?$" "~/.emacs" "^_emacs\\(\\.elc?\\)?$" (initialization "`_emacs' init file is deprecated, please use `.emacs'") "~/_emacs" t load expand-file-name "init" file-name-as-directory "/.emacs.d" file-name-extension "elc" file-name-sans-extension ".el" file-exists-p file-newer-than-file-p message "Warning: %s is newer than %s" sit-for 1 "default"] 7]()
  command-line()
  normal-top-level()
```

**Comment by hostmaster** [comment]
I've found a simular issue in `powerline` https://github.com/milkypostman/powerline/issues/40 and the fix is seems to be here https://github.com/milkypostman/powerline/commit/240bad5ebea4c7298e86ed76bd44c45d9062cb33

**Comment by sellout** [comment]
Fixed by #137.
